src/journal/journald-gperf.gperf

```diff
@@ -22,6 +22,7 @@ Journal.Storage,            config_parse_storage,    0, offsetof(Server, storage
 Journal.Compress,           config_parse_compress,   0, offsetof(Server, compress)
 Journal.Seal,               config_parse_bool,       0, offsetof(Server, seal)
 Journal.ReadKMsg,           config_parse_bool,       0, offsetof(Server, read_kmsg)
+Journal.BootKMsg,           config_parse_bool,       0, offsetof(Server, boot_kmsg)
```

Comment:
this shouldn't be a new option, but maybe a new enum value of ReadKMsg=. But I am not sure this is a really convincing feature. I mean, i am not sure it's a good option to just waste the early boot msgs entirely. It's a cheap hack, cutting corners, not an actual fix for possible performance issues...